.travis.yml

```diff
@@ -43,6 +43,21 @@ matrix:
         - python: 3.5
           env: TOX_ENV=py35-django111
 
+        - python: 3.6
+          env: TOX_ENV=py36-django18
+
+        - python: 3.6
+          env: TOX_ENV=py36-django19
+
+        - python: 3.6
+          env: TOX_ENV=py36-django110
+
+        - python: 3.6
+          env: TOX_ENV=py36-django111
+
+        - python: 3.6
+          env: TOX_ENV=py36-django20
+
```

Comment:
Not sure we have to test every combination of Python and Django, as long as we test each version once.